.github/workflows/docker.yml

```diff
@@ -6,19 +6,21 @@ on:
     paths:
     - '**Dockerfile'
     - '**docker.yml'
-#  workflow_dispatch:
-#  schedule:
-#    - cron: '30 11 20 */3 *'
+  pull_request:
+    branches: master
+    paths:
+    - '**Dockerfile'
+    - '**docker.yml'
+  workflow_dispatch:
+  schedule:
+    - cron: '25 1 5 * *'
 
 jobs:
   build-env-image:
     runs-on: ubuntu-24.04
+    env:
+      REGISTRY_IMAGE: ghcr.io/andy5995/vs-fltk
```

Comment:
This worked on my fork https://github.com/andy5995/vs-fltk/pkgs/container/vs-fltk